website/source/docs/configuration/storage/postgresql.html.md

```diff
@@ -42,6 +42,19 @@ CREATE TABLE vault_kv_store (
 CREATE INDEX parent_path_idx ON vault_kv_store (parent_path);
 ```
 
+Store for HAEnabled backend
+
+```sql
+CREATE TABLE vault_ha_store (
+  ha_key                                      TEXT COLLATE "C" NOT NULL,
+  ha_identity                                 TEXT COLLATE "C" NOT NULL,          
+  ha_value                                    TEXT COLLATE "C",  
+  valid_until                                 TIMESTAMP NOT NULL,
```

Comment:
In my experience TIMESTAMP WITHOUT TIME ZONE has been been a source of bugs.  Could you add WITH TIME ZONE here and in the test?

Reply:
I trust your experience here, seems like a source for bugs.